src/backend/src/services/ws/index.js

```diff
@@ -98,4 +99,18 @@ export default class WebSocketManager {
     });
     return players;
   }
+
+  findGame(ws) {
+    // Find a game that hasn't started yet
+    const gamesInLobby = [];
+    this.games.forEach((value, key) => {
+      if (value.round === 0) {
+        gamesInLobby.push(key);
+      }
+    });
+    const rand = Math.floor(Math.random() * gamesInLobby.length);
+    ws.send(
+      JSON.stringify({ method: Events.FIND_GAME, gameID: gamesInLobby[rand] })
```

Comment:
Possible concurrency problems, games in filtered list could start before/while being chosen with random.